cocos/profiler/profiler.ts

```diff
@@ -293,31 +306,84 @@ export class Profiler extends System {
         const rowHeight = height / this._totalLines;
         const lWidth = height / this._wordHeight;
         const scale = rowHeight / _constants.fontSize;
-        const columnWidth = this._eachNumWidth * this._canvas!.width * scale;
+        const columnWidth = this._eachNumWidth * canvas.width * scale;
+
+        const bgRight = lWidth + _constants.segmentsPerLine * columnWidth;
+        const bgPadding = columnWidth;
+
         const vertexPos: number[] = [
-            0,      height, 0, // top-left
-            lWidth, height, 0, // top-right
-            lWidth,      0, 0, // bottom-right
-            0,           0, 0, // bottom-left
+            -bgPadding, height + bgPadding, 0, // top-left
+            bgRight + bgPadding, height + bgPadding, 0, // top-right
+            bgRight + bgPadding, -bgPadding, 0, // bottom-right
+            -bgPadding, -bgPadding, 0, // bottom-left
```

Comment:
Add the background vertex at the first of vertices  since background should be drawn firstly.